### PR TITLE
Calculator: Milestone1

### PR DESCRIPTION
**Pull request by mahmoud717**
- create a react app.
- setup linters.
- published to Heroku
- published to Github.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @mahmoud717  👋

**This looks good. 👌**

Let's make just a few changes:

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by mikenath223** [comment]
- [ ] Please follow the git flow rule. What this means is you should make the `development` branch the default branch.

File: src/App.css [hunk 1]
```diff
@@ -32,6 +32,7 @@
   from {
     transform: rotate(0deg);
   }
+
```
Comment (mikenath223) [inline src/App.css]: - [ ] Most of the styles here are unnecessary. Please remove them or the App.css file itself.

File: src/App.js [hunk 2]
```diff
@@ -7,7 +7,11 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
```
Comment (mikenath223) [inline src/App.js]: - [ ] Replace the code here with a default line perhaps `<h1>Hello World</h1>` would do for the meantime.

File: src/index.css [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
```
Comment (mikenath223) [inline src/index.css]: - [ ] You also don't need this style rule here. Kindly remove it.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @mahmoud717 :hand: 
Awesome job on this milestone and also applying the requested changes.
However there still some changes we need to make before moving on.

- [ ] We need to add a descriptive README file into your repository. Right now, your README file it's the one generated by `create-react-app` and lacks information about the Calculator project.
Screenshot from 2021-01-13 10-03-56



- [ ] We also need to deploy the application into the web (Heroku, Netlify, etc) and add the live link to the README file.
Screenshot from 2021-01-13 10-01-38

Feel free to message me on Slack if you have any question :smile: 
Happy coding :computer: 
**Sergio Torres**

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**